app/src/main/AndroidManifest.xml

```diff
@@ -24,7 +26,51 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data
+                    android:scheme="tasky://"
+                    android:host="www.myapp.com"
```

Comment:
Why this host? You can just set it to `eventScreen` or so and leave the path prefix